comics/aggregator/crawler.py

```diff
@@ -298,3 +298,23 @@ def crawl_helper(self, feature_id, pub_date):
                 page = self.parse_page(release['url'])
                 url = page.src('img[itemprop="image"]')
                 return CrawlerImage(url)
+
+
+class NettserierCrawlerBase(CrawlerBase):
+    """Base comics crawler for all comics posted at nettserier.no"""
+    time_zone = 'Europe/Oslo'
+
+    def crawl_helper(self, short_name, pub_date):
+        page_url = 'https://nettserier.no/%s/' % (short_name)
+        page = self.parse_page(page_url)
+        site_date = page.text('p[class="comic-pubtime"]')
+        f = 'Published %Y-%m-%d %H:%M:%S'
+        date = datetime.datetime.strptime(site_date, f)
```

Comment:
I think it is better if either:

-`f` is inlined and the line broken in two after `strptime(`, or
- `f` is renamed `format`.